core/templates/dev/head/pages/exploration_editor/settings_tab/settings_tab.html

```diff
@@ -87,8 +87,46 @@
               </div>
             </div>
           </div>
-        </div>
+          <div class="text-right">
+            <button class="btn btn-success" style="width: 15em;" data-toggle="modal" data-target="#myModal" onclick="load()">Preview Summary Tite</button>
+          </div>
+          <div id="myModal" class="modal fade" role="dialog">
+            <div class="modal-dialog">
+              <div class="modal-content">
+                <div class="modal-header" style="height: 65px;">
+                  <button type="button" class="close" data-dismiss="modal">&times;</button>
+                  <h3 class="text-center" style="padding-top: 0px;">Summary Tile</h3>
+                </div>
+                <div class="modal-body text-center">
+                  <md-card class="oppia-activity-summary-tile oppia-dashboard-card-view-item protractor-test-exploration-dashboard-card md-default-theme">
+                    <a ng-href="/create/3f0fkDxYOqAd" href="/create/3f0fkDxYOqAd">
+                      <div class="title-section" style="background-color: #a33f40;">
+                        <img class="thumbnail-image" ng-src="/assets/images/subjects/Lightbulb.svg" src="/assets/images/subjects/Lightbulb.svg">
```

Comment:
Ditto. In fact, why are we repeating everything here from the activity summary tile? Rather, this should reuse the existing activity summary tile directive in some way. That way, if the design of that directive is updated, this will also update automatically and not fall out of sync.